devices/atlantic.js

```diff
@@ -0,0 +1,127 @@
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === 5) {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 1, type: 0x10}});
+            } else {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 0, type: 0x10}});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4273: {value: index, type: 0x04}});
+            }
+            return {state: {ac_louver_position: value}};
+        },
+    },
+    energy_mode: {
+        key: ['energy_mode'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(energyMode));
+            const index = energyMode[value];
+            switch (index) {
+            case energyMode.eco:
+                await entity.write('hvacThermostat', {'programingOperMode': 4});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}});
+                break;
+            case energyMode.normal:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacThermostat', {0x4270: {value: 0, type: 0x10}});
+                break;
+            case energyMode.powerfull:
+                await entity.write('hvacThermostat', {'programingOperMode': 0});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacThermostat', {0x4270: {value: 1, type: 0x10}});
+                break;
+            }
+            return {state: {energy_mode: value}};
+        },
+    },
+};
+
+module.exports = [{
+    zigbeeModel: ['Adapter Zigbee FUJITSU'],
+    model: 'GW003-AS-IN-TE-FC',
+    vendor: 'Atlantic Group',
+    description: 'Interface Naviclim for Takao air conditioners',
+    fromZigbee: [
+        fz.thermostat,
+        fz.fan,
+    ],
+    toZigbee: [
+        tzLocal.quiet_fan,
+        tzLocal.ac_louver_position,
+        tzLocal.energy_mode,
+        tz.thermostat_local_temperature,
+        tz.fan_mode,
+        tz.thermostat_occupied_heating_setpoint,
+        tz.thermostat_occupied_cooling_setpoint,
+        tz.thermostat_system_mode,
+        tz.thermostat_programming_operation_mode,
+        tz.thermostat_weekly_schedule,
+    ],
+    exposes: [
+        e.programming_operation_mode(),
```

Comment:
I don't know why, but in the front end for this option, I need to press the rounded arrow for it to update:
1. click on `setpoint`
2. click on `schedule`
3. `setpoint` stay highlighted.

Reply:
For it to update directly after the command, you need to return the new state in the `toZigbee.convertSet()` method